Palo Alto Networks/paloalto/_meta/fields.yml

```diff
@@ -0,0 +1,24 @@
+action.target:
+  description: 'Target action'
+  name: action.target
+  type: keyword
+geo.country_iso_code:
+  description: "Country ISO code"
+  name: geo.country_iso_code
+  type: keyword
+http.response.content_type:
+  description: "Http reponse content type"
+  name: http.response.content_type
+  type: keyword
+action.properties.subject:
+  description: "Subject"
+  name:  action.properties.subject
+  type: keyword
+email.from:
+  description: "Email address from"
+  name: email.from
+  type: keyword
+email.to:
+  description: "Email address to"
+  name: email.to
+  type: keyword
```

Comment:
Those fields should be prefixed with the product name

Reply:
For new ECS fields coming in next releases, it seems OK to declare them here